src/deploy/functions/prepare.ts

```diff
@@ -168,7 +151,7 @@ export async function prepare(
   );
 
   // Validate the function code that is being deployed.
-  validate.functionIdsAreValid(backend.allEndpoints(wantBackend));
+  functionIdsAreValid(backend.allEndpoints(wantBackend));
```

Comment:
See... this is why I liked the package name. validate.functionIdsAreValid is a command. functionIdsArevalid sounds like it should return a bool. If we're not going to use the package name can we rename the function something like `validateFunctionIds` or `assertFunctionIdsAreValid`?

Reply:
Reverted.